ts/Core/Time.ts

```diff
@@ -954,6 +957,21 @@ class Time {
  * */
 
 namespace Time {
+    export interface DateTimeLabelFormatObject {
+        from?: string;
+        list?: string[];
+        main: string;
+        range?: boolean;
+        to?: string;
+    }
+    export type DateTimeLabelFormatOption = (
+        string|
+        Array<string>|
+        Time.DateTimeLabelFormatObject
+    );
+    export type DateTimeLabelFormatsOption = (
+        Record<TimeUnit, DateTimeLabelFormatOption>
+    );
```

Comment:
The naming could be simplified as the Time class has a DTL function, so DTLFormatObject etc.

Reply:
I don't know... The name of the internal `resolveDTLFormat` function is an attempt to save filesize, but for the typing I think we should spell it out.